styles.css

```diff
@@ -662,4 +659,136 @@ body {
   .navMenu.active {
     left: 0;
   }
+
+  .insideDiv {
```

Comment:
@mwanawabangona You are comparing the tablet view with the Mobile version in Figma so it won't look exactly the same since you want to see changes so I would make it look as required.

Reply:
@mwanawabangona Yes, we can.

Reply:
@waseemqureshi2020 Thank you for taking the call with me.